backend/database.py

```diff
@@ -0,0 +1,71 @@
+import datetime
+import psycopg2
+
+import config
+
+INSERT_PROMPT_CMD = """
+    INSERT INTO prompts (timestamp, prompt, seed, nsfw_text, nsfw_image)
+    VALUES (%(timestamp)s, %(prompt)s, %(seed)s, %(nsfw_text)s, %(nsfw_image)s) 
+    RETURNING prompt, timestamp;
+"""
+
+
+def create_connection():
+    return psycopg2.connect(
+        database=config.DB_NAME, user=config.DB_USER, password=config.DB_PASSWORD
```

Comment:
how come we don't need URL and port to connect to the db? Are these some smart defaults?

Reply:
The port is the default one for Postgres server - 5432. The default for host is described in the doc as "UNIX socket". My understanding of inter-process communication is not very deep, but from what I understand this means connecting with some other process on localhost.

I have added the host and port parameters for clarity.